RadixWallet/Core/Resources/Resources/en.lproj/Localizable.strings

```diff
@@ -1038,3 +1020,15 @@ If you have any “Legacy” Accounts (created on the Olympia network) to import
 "survey_reason_heading" = "What’s the main reason for your score?";
 "survey_reason_fieldHint" = "Let us know...";
 "survey_submitButton" = "Submit Feedback - Thanks!";
+"factorSourceAccess_title_signature" = "Signature Request";
```

Comment:
Are all of these new strings? if yes then those need to be added first with a pull request on https://github.com/radixdlt/apps-localization-src,
there is also a general guide https://radixdlt.atlassian.net/wiki/spaces/AT/pages/3241246746/Localization+process.

But in short, once that PR is merged, a PR like [this](https://github.com/radixdlt/babylon-wallet-ios/pull/1071/files) will be created on the wallet repo, then swiftgen will need to be used to generate the L10n , merged in main, and after that you do rebase your branch.

Reply:
thanks for sharing, I didn't know we had this external `apps-localization-src`. In this case, there are some new Strings (content changed) and some which had the key used renamed. Do I need to sync with Android for this?

Reply:
Yes, as they will need to pull in the new strings and update their codebase.